app/v2/controllers/V2DeparturesController.scala

```diff
@@ -237,4 +238,8 @@ class V2DeparturesControllerImpl @Inject() (
             response => Ok(Json.toJson(HateoasDepartureIdsResponse(response)))
           )
     }
+
+  def sendMessageDownstream(departureId: DepartureId): Action[Source[ByteString, _]] = Action(streamFromMemory) {
```

Comment:
While this was the name given to the method in version 1 of the API, I'm not sure I like it. I think I'd prefer something like "attachMessage", rather than "sendMessageDownstream", which is only one part of what is going on here.

Reply:
done